bpfassets/libbpf/src/kepler.bpf.c

```diff
@@ -214,11 +206,16 @@ static unsigned int get_task_state(void *task)
 	return ((struct task_struct___o *)task)->state;
 }
 
+static unsigned int get_task_tgid(void *task)
```

Comment:
No need for a function here. With the change above you can just use `prev_task->tgid` directly.